data/strings/types_strings.txt

```diff
@@ -4434,7 +4338,8 @@
     uk = Поштомат
 
   [type.amenity.vending_machine.fuel]
-    en = Fuel Dispenser
+    en = Gas Pump
+    en-GB = Petrol Pump
```

Comment:
Based on fuel above, probably should just be Fuel.   In the US, "gas" is gasoline, vs diesel.  I am unclear on petrol being just gas or gas and diesel both.    But many people the US would call the device you use to put diesel in your car/truck a gas pump.

Reply:
changed to fuel pump